eo.json

```
@@ -305,24 +371,135 @@
-        "Indicator.ServerStatus": "<color=#fff>Servila stato:</color> {status, select, Good {Bona} Slow {Malrapida} Down {Malsupre} NoInternet {Neniu interreto}} ({response_ms} ms)",
+        "Indicator.OnlineUsers": "<color=hero.yellow>{users}</color> {users,plural, one {Enreta Uzanto} other {Enretaj Uzantoj}}",
+        "Indicator.OnlineUsersVR": "<color=hero.purple>{users}</color> {users, plural, one {uzanto} other {uzantoj}} en VR",
+        "Indicator.OnlineUsersDesktop": "<color=hero.cyan>{users}</color> {users, plural, one {uzanto} other {uzantoj}} en Labortabla",
+
+        "Indicator.ServerStatus.Status": "Servila stato",
+        "Indicator.ServerStatus.Good": "Bona",
+        "Indicator.ServerStatus.Slow": "Malrapida",
+        "Indicator.ServerStatus.Down": "Malsupre",
+        "Indicator.ServerStatus.NoInternet": "Neniu interreto",
 
         "Indicator.Sync.AllSynced": "Ĉiuj sinkronigitaj",
         "Indicator.Sync.SyncingItems": "Sinkronigante {item_count,plural, one {# ero} other {# erojn}} <nobr>({item_percent} %)",
         "Indicator.Sync.UploadingVariants": "Alŝutado de {variant_count,plural, one {# asset variant} other {# asset variants}}",
         "Indicator.Sync.SyncError": "Eraro de sinkronigo! <size=50%>Check log for details</size>",
-        "Indicator.Sync.OutOfSpace": "Ekstere de spaco! <size=50%>Cannot sync</size>",
+        "Indicator.Sync.StorageFull": "Konservejo plenas! <size=61.803%>Ne povas sinkronigi</size>",
+
+        "Indicator.Storage.StorageFull": "Konservejo Plenas",
+        "Indicator.Storage.Usage": "{storage_used} de {storage_max}<br><size=61.803%>de Space en la Konservejo Uzita</size>",
 
         "Indicator.LiveHiddenMessage": "(informoj kaŝitaj dum dissendado)",
 
+        "Login.InvalidCredentials": "Nevalidaj identigiloj",
+        "Login.InvalidCredentials.Description": "Kontroli viajn identigilojn kaj reprovi aŭ premi \"Pasvorto Perdita?\"",
+
+        "Login.InvalidLogin": "Nevalidaj identigiloj",
+        "Login.InvalidLogin.Description": "Kontroli viajn identigilojn kaj reprovi aŭ premi \"Pasvorto Perdita?\"",
+
+        "Login.MalformedUID": "Nevalida aparatinformo",
+        "Login.MalformedUID.Description": "Bonvolu provi reinstali, aŭ kontakti subtenadon je support.resonite.com.",
+        
+        "Login.MissingUID": "La aparatinformo mankas",
+        "Login.MissingUID.Description": "Bonvolu provi reinstali, aŭ kontakti subtenadon je support.resonite.com.",
+        
+        "Register.MissingUID": "La aparatinformo mankas",
+        "Register.MissingUID.Description": "Bonvolu provi reinstali, aŭ kontakti subtenadon je support.resonite.com.",
+        
+        "Register.MalformedUID": "Nevalida aparatinformo",
+        "Register.MalformedUID.Description": "Bonvolu provi reinstali, aŭ kontakti subtenadon je support.resonite.com.",
+
+        "Login.MissingLogin": "Neniuj indentigiloj provizataj",
+        "Login.MissingLogin.Description": "Ĉu vi forgesis provizi itentigilojn?",
```

Oni provizas iun per io.
```suggestion
        "Login.MissingLogin": "Mankas identigilo",
        "Login.MissingLogin.Description": "Ĉu vi forgesis doni identigilojn?",
```

---

Mi kredas ke "provizi" ne estas malĝuste (per PIV, ĉi tie: https://vortaro.net/#provizi_kdc), sed mi tre preferas "mankas", dankon!